aten/src/ATen/native/mkldnn/xpu/BlasImpl.h

```diff
@@ -0,0 +1,518 @@
+#include <ATen/ATen.h>
```

Comment:
should we merge this file into Blas.cpp to align with pytorch?

Reply:
`BlasImpl.h` would be used more than `Blas.cpp`. It would also be used for in `xpu/Linear.cpp` for linear pointwise fusion, it's more like a common utils. I recommend we may keep it :)

Reply:
Please update your PR description to describe the relationship between `BlasImpl.h` and `Blas.cpp`

Reply:
What functions are being used by `Blas.cpp`? I might miss something but couldn't find the usage there.

Reply:
@jgong5  Thanks for the reminding.  The BlasImpl.h is not used in Blas.cpp. I'd move it into Linear fusion pr